src/main/java/com/example/booksstore/controller/BookController.java

```diff
@@ -32,4 +34,14 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestDto bookDto) {
+        return bookService.updateById(id, bookDto);
+    }
+
+    @DeleteMapping("/{id}")
```

Comment:
add `no content` response status to here to let user know that it is expected to not have a response body, and `created` for the post endpoint that you already have